changelog/0.46.md

```diff
@@ -0,0 +1,73 @@
+# v0.46.0
+
+Released 2025-05-07
+
+> [!WARNING]
+> **Security Notice(s)**
+>
+> - Ingress-nginx upgraded to address the following CVEs:<br>CVE-2025-24513<br>CVE-2025-24514<br>CVE-2025-1097<br>CVE-2025-1098<br>CVE-2025-1974
+<!-- -->
+> [!IMPORTANT]
+> **Platform Administrator Notice(s)**
+>
+> - Default Opensearch setup has been changed. Opensearch is now configured to use 3 master nodes, 0 data nodes and 0 client nodes by default. There's no easy migration path for the new configuration, if the old default Opensearch setup is used it should be added to the override config.<br>Proxy protocol is now enabled by default in ingress-nginx when using ElastX as infrastructure provider.<br>Fluentd audit is now enabled by default when using prod or air-gapped flavor.
+> - The GPU Operator has received some new configuration options in order to make it easier to deploy in a multi-tenant way
+> - A new gatekeeper policy has been added that will deny any PodDisruptionBudget and connected Pod controller if the PodDisruptionBudget does not allow at least 1 Pod disruption. Note that this will apply in both sc and wc, it will also apply to namespaces even if they have the label owner=operator.
+> - The requirements have moved to Ubuntu 24.04 LTS (Noble Numbat).<br>Additionally many requirements have been updated which can be installed using the `ck8s install-requirements` command.
+> - [!note]<br>From this point onward the requirements installation step will install yq version 4 as simply `yq`.<br>Most repositories have been updated to work with this change in mind, but if you've got any other workflows depending on `yq` being version 3 and that are running in the context of this repository's docker image or wrapper scripts, please inform someone from @elastisys/goto-scripts so we can update them too. This situation will be easy to notice if you're running through a wrapper script as it will simply abort with the notice: `expecting the yq binary to be at least version v4`.
```

Comment:
```suggestion
> - From this point onward the requirements installation step will install yq version 4 as simply `yq`.<br>Most repositories have been updated to work with this change in mind, but if you've got any other workflows depending on `yq` being version 3 and that are running in the context of this repository's docker image or wrapper scripts, please inform someone from @elastisys/goto-scripts so we can update them too. This situation will be easy to notice if you're running through a wrapper script as it will simply abort with the notice: `expecting the yq binary to be at least version v4`.
```